b9/include/b9/Module.hpp

```diff
@@ -26,17 +25,7 @@ struct FunctionDef {
         instructions{instructions},
         nparams{nparams},
         nlocals{nlocals} {}
```

Comment:
I think we can actually get rid of this constructor too, and just rely on the implicit constructor.